components/mediaLibrary.html

```diff
@@ -78,7 +78,7 @@
   </div>
 
   <!-- Media Library -->
-  <div ng-if="$ctrl.showLibrary" class="media-library-content">
+  <div ng-if="$ctrl.showLibrary" class="media-library-content" ng-if="collection.models.length > 0 ">
```

Comment:
There should only be 1 `ng-if` per tag.

Reply:
am working on this today and will update you..